test/test-cases/regression/action-ctl_rule_remove_by_tag.json

```diff
@@ -34,7 +34,7 @@
       "debug_log":"Skipped rule id '2'. Skipped due to a ruleRemoveByTag action."
     },
     "rules":[
-      "SecRule ARGS:key \".\" \"id:4,ctl:ruleRemoveByTag=tag123",
+      "SecRule ARGS:key \".\" \"id:4,ctl:ruleRemoveByTag=tag123\"",
```

Comment:
This should not have worked before. It should have been a parsing error. Maybe open a Bug?

Reply:
> This should not have worked before. It should have been a parsing error.

Yes, definitely.

> Maybe open a Bug?

Unfortunately the parser has several issues. I don't know what would be the best step: open a new issue for each discovered (and already known) bug, or collect them into one issue? Or open a new one and collect the others there? (There are still few opened issue related to parser).

Btw I mark this review as resolved, actually it does not affect this PR.